packages/stryker-vue-mutator/src/VueMutator.ts

```diff
@@ -0,0 +1,80 @@
+import { Config } from 'stryker-api/config';
+import { File } from 'stryker-api/core';
+import { Mutant, Mutator, MutatorFactory } from 'stryker-api/mutant';
+const compiler = require('vue-template-compiler');
+
+export default class VueMutator implements Mutator {
+  private mutators: { [name: string]: Mutator; };
+
+  constructor(private config: Config) { }
+
+  mutate(inputFiles: File[]): Mutant[] {
+    this.initializeMutators();
```

Comment:
Can't we move this to the constructor. That way it works nicely with TS 2.7 `--strictPropertyInitialization`

Reply:
Fixed